lib/libs/email/preview/Initial Submissions/CMS/CHIP_SPA.tsx

```diff
@@ -1,6 +1,6 @@
-import { ChipSpaCMSEmail } from "lib/libs/email/content/new-submission/emailTemplates/ChipSpaCMS";
-import { emailTemplateValue } from "lib/libs/email/mock-data/new-submission";
-import * as attachments from "lib/libs/email/mock-data/attachments";
+import { ChipSpaCMSEmail } from "libs/email/content/new-submission/emailTemplates/ChipSpaCMS";
+import { emailTemplateValue } from "libs/email/mock-data/new-submission";
+import * as attachments from "libs/email/mock-data/attachments";
```

Comment:
Vitest doesnt like when we do lib/libs.  You will see this fixed in a couple places.